app/src/main/res/menu/side_navigation_bar.xml

```diff
@@ -20,10 +20,6 @@
         <item android:title="@string/profile">
             <menu>
 
-                <item
-                    android:id="@+id/nav_login"
-                    android:icon="@drawable/ic_login_icon"
-                    android:title="@string/login" />
```

Comment:
This will result in a conflict with the #258 issue since users will no longer need to sign in again if they are already signed in, and thus this button will do nothing but just redirection to the main page. This is a good point and the button must be kept on the custom main page for the version where the sign-in is no longer needed since, correct me if I'm wrong, there will be a completely new main activity with only features that don't need an internet connection, here it's relevant to add this button in order to enable the user to go back to the online mode.